src/shared/tpm2-util.c

```diff
@@ -382,6 +401,21 @@ int tpm2_context_new(const char *device, Tpm2Context **ret_context) {
         if (r < 0)
                 return r;
 
+        /* We require AES and CFB support for session encryption. */
+        if (!tpm2_supports_alg(context, TPM2_ALG_AES))
+                return log_error_errno(SYNTHETIC_ERRNO(ENOTRECOVERABLE), "TPM does not support AES.");
+        if (!tpm2_supports_alg(context, TPM2_ALG_CFB))
+                return log_error_errno(SYNTHETIC_ERRNO(ENOTRECOVERABLE), "TPM does not support CFB.");
+
+        context->session_symmetric_template = (TPMT_SYM_DEF){
```

Comment:
usually we'd place a space between `)` and `{`

Reply:
ack